django-stubs/db/models/fields/reverse_related.pyi

```diff
@@ -77,6 +78,9 @@ class ForeignObjectRel(FieldCacheMixin):
         ordering: Sequence[str] = (),
     ) -> _ChoicesList: ...
     def is_hidden(self) -> bool: ...
+    @deprecated(
+        "The get_joining_columns() is deprecated and will be removed in Django 6.0. Use get_joining_fields() instead."
+    )
```

Comment:
To be consistent, the "The" prefix (and in other places) should probably be removed.